README.md

```diff
@@ -11,160 +8,142 @@
 
-## Lead Maintainer
+## Lead maintainer
 
-[Marcin Rataj](https://github.com/lidel)
+[Marcin Rataj](https://github.com/lidel), with help from the [IPFS GUI working group](https://github.com/ipfs/ipfs-gui).
 
-## Table of Contents
+## Table of contents
 
-- [Background](#background)
+- [About IPFS Companion](#about-ipfs-companion)
 - [Features](#features)
-- [Install](#install)
+- [Install](#install-ipfs-companion)
 - [Contribute](#contribute)
-- [Troubleshooting](#troubleshooting)
-- [Privacy Policy](#privacy-policy)
-- [License](#license)
+- [Help & Troubleshooting](#help--troubleshooting)
+- [Privacy & Licenses](#privacy--license)
 
-## Background
+## About IPFS Companion
 
-This add-on enables everyone to access IPFS resources the way they were meant: from locally running IPFS node :-)
+IPFS Companion harnesses the power of your locally running IPFS node (either through the IPFS Desktop app or the command-line daemon) directly inside your favorite Chromium-based or Firefox browser, enabling support for `ipfs://` addresses, automatic IPFS gateway loading of websites and file paths, easy IPFS file import and sharing, and more.
 
-IPFS is a new hypermedia distribution protocol, addressed by content and identities.
-IPFS enables the creation of completely distributed applications.
-It aims to make the web faster, safer, and more open.
+IPFS is a peer-to-peer hypermedia protocol designed to make the web faster, safer, more resilient, and more open. It enables the creation and dissemination of completely distributed sites and applications that don’t rely on centralized hosting and stay true to the original vision of an open, flat web. Visit [the IPFS Project website](https://ipfs.io) to learn more.
 
-Learn more at [ipfs.io](https://ipfs.io) (it is really cool, we promise!)
+## IPFS Companion features
 
-## Features
+### Automatically detect and redirect IPFS resources
 
-### Automagical Detection of IPFS Resources
+#### Detect URLs with IPFS paths
 
-#### IPFS Path in URL
-
-Requests for IPFS-like paths (`/ipfs/{cid}` or `/ipns/{peerid_or_host-with-dnslink}`) are detected on any website.  
-If tested path is a [valid IPFS address](https://github.com/ipfs/is-ipfs) it gets redirected and loaded from a local gateway, e.g:  
+IPFS Companion detects and tests requests for IPFS-like paths ( such as `/ipfs/{cid}` or `/ipns/{peerid_or_host-with-dnslink}`) on any website. If a path is a [valid IPFS address](https://github.com/ipfs/is-ipfs), it is redirected to load from your local gateway:  
 > `https://ipfs.io/ipfs/QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR`  
 > → `http://127.0.0.1:8080/ipfs/QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR`
```

Comment:
We switched the default from IP to hostname to enable subdomain support, examples should reflect that:
```suggestion
> → `http://localhost:8080/ipfs/QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR`
> → `http://bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi.ipfs.localhost:8080`
```

Reply:
Oh drat -- thanks for catching that.